src/plugins/ssh/index.ts

```diff
@@ -55,22 +45,13 @@ const DESTINATION_READ_ERROR =
   /Error while connecting \[4010: 'destination read failed'\]/;
 const GOOGLE_LOGIN_MESSAGE =
   /You do not currently have an active account selected/;
+const SUDO_MESSAGE = /Sorry, user .+ may not run sudo on .+/; // The output of `sudo -v` when the user is not allowed to run sudo
```

Comment:
Is this universal across unix-style operating systems? Or is this just validated on a single version of a single OS?

Reply:
I only validated on one operating system. Should behave the same way? It's part of the sudo command itself.